packages/components/src/internal/components/domainproperties/PropDescType.ts

```diff
@@ -251,6 +252,19 @@ export const AUTOINT_TYPE = new PropDescType({
     alternateRangeURI: 'xsd:int',
 });
 
+export const VISIT_DATE_TYPE = new PropDescType({
+    name: 'visitDate',
+    display: 'VisitDate',
```

Comment:
This was probably decided in the spec, but any reason not to make this display text a little more human readable? Suggestions: "Visit Date" vs "Visit ID", "Visit (Date)" vs "Visit (ID)", "Timepoint (date based)" vs "Timepoint (visit based)"
@labkey-klum Thoughts?

Reply:
Yes, I like that idea maybe even : Timepoint (Date event) and Timepoint (Visit event)...
Probably the least controversial nomenclature would be "Visit Date" and "Visit ID". Maybe we go for that and then slot in some time at the end of the epic for some UI polish with the product managers and designers?

Reply:
Done! Also made a note in the 'Support Link to Study in Sample Types' spec in 'Miscellaneous Wrap Up' to remind about the wording finalization task for new UI elements, with product managers and designers, as Karl suggests